compose/cli/main.py

```diff
@@ -144,6 +144,8 @@ class TopLevelCommand(DocoptCommand):
       -f, --file FILE           Specify an alternate compose file (default: docker-compose.yml)
       -p, --project-name NAME   Specify an alternate project name (default: directory name)
       --verbose                 Show more output
+      --project-directory PATH  Specify an alternate working directory
+                                (default: the path of the compose file)
```

Comment:
hmm, I think this should say "the directory of the compose file" or "the dirname of the compose file".